en/plugins/zapier.md

```diff
@@ -63,12 +81,9 @@ At this point choose which Trigger or Action you need. Each trigger will get you
 
 ### 3. Authorize Mautic instance
 
-Once you choose to use Mautic integration you'll need to authorize your Mautic to it. Mautic use basic auth as mentioned earlier. So all you need is a Mautic user credentials and URL of where your Mautic lives. It's recommended to create a new user for Zapier which will have some advantages:
-
-1. Giving a third party app credentials to your Mautic is a security risk. If something happens you simply delete this special Zapier user and your admin user will be safe.
-2. You will see what contacts were created by Zapier simply by looking at the created by user.
+Once you choose to use Mautic integration you'll need to authorize your Mautic to it. 
```

Comment:
Once you choose to enable the integration, you will need to authorize Mautic to communicate with Zapier.